### PR TITLE
created new component for teacher to view rumble as table

### PR DESCRIPTION
**Pull request by LizDrumm**

### CODE REVIEW

**Review by LizDrumm — COMMENTED** [review]
Ignore this comment- not sure how to delete it.

**Review by LizDrumm — COMMENTED** [review]
Is this wrong? I feel like it is- but it was how I got the end_time to work on the `SectionRumbleCard`

File: src/components/pages/TeacherDashboard/TeacherViewSection/SectionRumbleList/SectionRumbleCard.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  const [status] = useRumbleStatus(rumble.phase);
+
+  return (
+    <div>
+      <div>
+        <Table.Row>
+          <Table.Col>
+            {newMonth + 1}/{newDay}/{newYear}
+          </Table.Col>
+          {status !== 'Complete' && <Table.Col>{rumble.phase}</Table.Col>}
```
Comment (bramirez96) [inline src/components/pages/TeacherDashboard/TeacherViewSection/SectionRumbleList/SectionRumbleCard.tsx]: Simplify this with a ternary

File: src/components/pages/TeacherDashboard/TeacherViewSection/SectionRumbleList/SectionRumbleCard.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  const openRumble = () => {
+    setCurrentSection(section);
+    setCurrentRumble(rumble);
+    clearCurrentStudent();
+    push('/dashboard/teacher/rumble', { rumble, section });
+  };
+
+  //Setting date and day of the week
+  const newDate = new Date(rumble.end_time);
```
Comment (bramirez96) [inline src/components/pages/TeacherDashboard/TeacherViewSection/SectionRumbleList/SectionRumbleCard.tsx]: Don't declare stateless variables in the scope of a component if possible, use Luxon

File: src/api/Rumbles/rumbleTypes.ts [hunk 3]
```diff
@@ -6,7 +6,7 @@ export interface IRumbleWithSectionInfo extends IRumble {
 export interface IRumble extends INewRumble {
   id: number;
   created_at: Date;
-  end_time?: Date;
+  end_time: Date;
```
Comment (bramirez96) [inline src/api/Rumbles/rumbleTypes.ts]: Put it back for inactive phase

File: src/components/pages/TeacherDashboard/TeacherViewSection/SectionStudentList/SectionStudentListContainer.tsx [hunk 4]
```diff
@@ -18,8 +18,8 @@ const SectionStudentListContainer = ({
   });
 
   useEffect(() => {
-    getStudents(section.id);
-  }, [section]);
+    if (!studentList && !isLoading) getStudents(section.id);
```
Comment (bramirez96) [inline src/components/pages/TeacherDashboard/TeacherViewSection/SectionStudentList/SectionStudentListContainer.tsx]: Can we check if this loads the correct students for the class? I think I was wrong we might need to revert this

**Review by bramirez96 — CHANGES_REQUESTED** [review]

**Review by bramirez96 — APPROVED** [review]

**Pull request merged**